backend/src/users/auth/auth.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@nestjs/common';
+import { UsersService } from '../users.service';
+import * as refresh from 'passport-oauth2-refresh';
+import { PrismaService } from '../../prisma/prisma.service';
+import { Strategy } from 'passport-42';
+import { Response } from 'express';
+
+@Injectable()
+export class AuthService {
+	constructor(
+		private prismaService: PrismaService,
+	) {}
+
+	refreshToken(refreshToken: string) {
+		const prisma = this.prismaService;
+
+		refresh.use(
+			'refresh-42', 
+			new Strategy({
+				clientID: process.env.FT_UID,
+				clientSecret: process.env.FT_SECRET,
+				callbackURL: process.env.FT_CB,
+			}, function() {} )
+		);
+
+		refresh.requestNewAccessToken(
+			'refresh-42', 
+			refreshToken, 
+			async (err, newAccessToken, newRefreshToken, results) => {
+				const atExpiresAt = new Date((results.created_at + results.expires_in) * 1000);
+				const rtExpiresAt = new Date((results.created_at + results.expires_in * 84) * 1000);
```

Comment:
```suggestion
        const ONESECOND = 1000;
				const atExpiresAt = new Date((results.created_at + results.expires_in) * ONESECOND);
				const rtExpiresAt = new Date((results.created_at + results.expires_in * 84) * ONESECOND);
```

```suggestion
				const atExpiresAt = new Date((results.created_at + results.expires_in) * 1000); // 1 second
				const rtExpiresAt = new Date((results.created_at + results.expires_in * 84) * ONESECOND);
```

상수나 주석으로 기간 읽기 쉽게 표시하면 좋을 것 같습니다